types/openlayers/index.d.ts

```diff
@@ -14618,6 +14618,7 @@ declare module olx {
             maxZoom?: number;
             duration?: number;
             easing?: ((t: number) => number);
+            callback?: ((animationCompleted?: boolean) => void);
```

Comment:
Per the OL docs and source, the callback will always be called with a value. I don't see a situation where it would be optional.

Reply:
True, OL will pass `true` or `false` as the first argument to the `callback` function. But the callback function might have no parameters if it doesn't care about that value. If I make `animationCompleted` required, then that would throw an error. Or maybe I'm wrong -- would `callback` be allowed to have no parameters even if `animationCompleted` is required?

Reply:
Nevermind, I was wrong!
Also, I found an existing `AnimateCallback` type, so I used that instead of hardcoding it.